File: 하루에 한개씩 문제 풀기/Go/Programmers/Level 0/A 강조하기.go
import (
    "strings"
    "unicode"
)

func solution(myString string) string {
	result := strings.Builder{}
	for _, char := range myString {
		if char == 'a' {
			result.WriteString("A")
		} else if unicode.IsUpper(char) && char != 'A' {
			result.WriteString(strings.ToLower(string(char)))
		} else {
			result.WriteString(string(char))
		}
	}
	return result.String()
}

File: 하루에 한개씩 문제 풀기/Java/Programmers/Level 0/A 강조하기.java
class Solution {
    public static String solution(String myString) {
        StringBuilder result = new StringBuilder();
        for (char c : myString.toCharArray()) {
            if (c == 'a') {
                result.append('A');
            } else if (Character.isUpperCase(c) && c != 'A') {
                result.append(Character.toLowerCase(c));
            } else {
                result.append(c);
            }
        }
        return result.toString();
    }
}